gravitee-plugin-core/src/test/java/io/gravitee/plugin/core/internal/PluginRegistryTest.java

```diff
@@ -150,13 +154,73 @@ public void startWithValidWorkspace_withOneDependencyWithoutAliasDisabled() thro
         Assert.assertTrue(pluginRegistry.plugins().stream().noneMatch(p -> p.id().equals("custom-plugin")));
     }
 
+    @Test
+    public void startWithValidWorkspace_withDuplication() throws Exception {
+        String path = "/io/gravitee/plugin/with-duplication/";
+
+        // we are using different version to assert which one is loaded
+        updateModifiedTimestampCustomPlugin2();
+
+        PluginRegistryImpl pluginRegistry = initPluginRegistry(path);
+        pluginRegistry.start();
+
+        Assert.assertEquals(1, pluginRegistry.plugins().size());
```

Comment:
```suggestion
        
        assertThat(pluginRegistry.plugins()).hasSize(1);
        assertThat(pluginRegistry.plugins()).allMatch(plugin -> {
            assertThat(plugin.id()).isEqualTo("custom-plugin");
            assertThat(plugin.manifest().version()).isEqualTo("2.0.0-SNAPSHOT");
            return true;
        });
```

Reply:
Didn't see that one: I've done this (better error message I think):
``` assertThat(pluginRegistry.plugins()).hasSize(1);
        assertThat(pluginRegistry.plugins()).first().extracting(Plugin::id).isEqualTo("custom-plugin");
        assertThat(pluginRegistry.plugins()).first().extracting(p -> p.manifest().version()).isEqualTo("2.0.0-SNAPSHOT");```